CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.3.0]
+RAiDER package was refactored to expose the main functionality as a Python library, including the `prepareWeatherModel` 
+and `tropo_delay` functions, as well as anciliarry functions needed for defining AOIs, look vectors, etc.
+
+### New/Updated Features
++ Python library access to main functions for accessing weather model data and calculating delays
++ Slant delay calculation through projection should be supported for all workflows
```

Comment:
Word “should” is not good to use. check and state what is supported.

Reply:
Updated to just include slant delays - projection on the cube, as promised from last release.